crates/schema/src/lib.rs

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2023 -  Restate Software, Inc., Restate GmbH.
```

Comment:
Just as curiosity, why the schema updater doesn't live here in this crate?

Reply:
The reason the schema updater does not live here is because it needs access to the `restate-service-protocol` crate for the `restate_service_protocol::discovery::schema` module. It is not a super big deal but that way I thought  that we could keep the `schema` crate a bit leaner dependency-wise and if we want we could move `restate-schema` into the `restate-types` crate.